libs/dh/metering-point/feature-overview/src/lib/secondary-master-data/dh-secondary-master-data.component.ts

```diff
@@ -75,13 +76,17 @@ export class DhSecondaryMasterDataComponent {
       meteringPoint?.productId ?? ''
     }`;
     const unit = `meteringPoint.unit.${meteringPoint?.unit ?? ''}`;
+    const netSettlementGroupAsNumber = `meteringPoint.secondaryMasterData.netSettlementGroupAsNumber.${
+      meteringPoint?.netSettlementGroup ?? ''
+    }`;
```

Comment:
suggestion: It feels strange to map the `netSettlementGroup` value to a translation string. Especially if the display value is always going to be a number (e.g. 6).

We can instead use a getter property and map the `netSettlementGroup` value to a number in there before displaying it on the page. For example:

```ts
  get netSettlementGroupAsNumber(): number | undefined {
    switch (this.#secondaryMasterData?.netSettlementGroup) {
      case NetSettlementGroup.Zero:
        return 0;
      case NetSettlementGroup.One:
        return 1;
      case NetSettlementGroup.Two:
        return 2;
      case NetSettlementGroup.Three:
        return 3;
      case NetSettlementGroup.Six:
        return 6;
      case NetSettlementGroup.NinetyNine:
        return 99;
      default:
        return;
    }
  }
```

Reply:
okay i will change it to this